docs/readme.md

```diff
@@ -1,23 +1,42 @@
 
 The Editor offers rich new value to users with visual, drag-and-drop creation tools and powerful developer enhancements with modern vendor packages, reusable components, rich APIs and hooks to modify and extend the editor through Custom Blocks, Custom Block Styles and Plugins.
 
-## Quick Links
+## Quick links
 
--   Beginners: The [Create a Block Tutorial](/docs/designers-developers/developers/tutorials/create-block/readme.md) walks through creating a block plugin using the `@wordpress/create-block` package; a quick and easy way to start creating your own block.
+### [Create a Block Tutorial](/docs/designers-developers/developers/tutorials/create-block/readme.md)
+Learn how to create your first block for the WordPress block editor. From setting up your development environment, tools, and getting comfortable with the new development model, this tutorial covers all what you need to know to get started with the block editor.
 
--   [Gutenberg Architecture](/docs/architecture/readme.md) covers the conceptual and tool choices going into the Gutenberg project, from repository structure to packages and tools.
+- [Dynamic Blocks](https://developer.wordpress.org/block-editor/tutorials/block-tutorial/creating-dynamic-blocks/)
 
--   [Block API Reference](/docs/designers-developers/developers/block-api/README.md)
+### [Develop for the block editor](https://developer.wordpress.org/block-editor/developers/)
+Whether you want to extend the functionality of the block editor, or create a plugin based on it, you will find here all the information about the basic concepts you need to get started, the block editor APIs and its architecture.
 
--   [Contributors Guide](/docs/contributors/readme.md) covers how **you** can help; be it code, design, documentation, language, or triage, we need you to help make Gutenberg.
+- [Block Filters](https://developer.wordpress.org/block-editor/developers/filters/block-filters/)
+- [Editor Filters](https://developer.wordpress.org/block-editor/developers/filters/editor-filters/)
+- [Theming for the Block Editor](https://developer.wordpress.org/block-editor/developers/themes/)
+- [Parser Filters](https://developer.wordpress.org/block-editor/developers/filters/parser-filters/)
+- [Block API Reference](https://developer.wordpress.org/block-editor/developers/block-api/)
+- [Block Editor Accessibility](https://developer.wordpress.org/block-editor/developers/accessibility/)
+- [Internationalization](https://developer.wordpress.org/block-editor/developers/internationalization/)
+
+
+
+<!-- Here we will have chapters on the API, component references, etc ...
+Also, we can have as bullet points links pointing to these chapters. -->
+
+### [Contribute to the block editor](/docs/contributors/readme.md)
```

Comment:
We might want to bring  back the architecture link removed above to this  section somehow.